core/rendered_connection.js

```diff
@@ -64,12 +64,20 @@ Blockly.RenderedConnection = function(source, type) {
    */
   this.offsetInBlock_ = new Blockly.utils.Coordinate(0, 0);
 
+  // TODO: This could be changed to an enum if you want.
```

Comment:
Yeah, that would be my preference. Easy to lose sight of what each value means throughout the code.

Reply:
Fixed. I added the enum as a property of Blockly.RenderedConnection, `Blockly.RenderedConnection.TrackedState`. Is there anyway that could be aliased? Or is there somewhere else it could be put? The verbosity makes the code look kinda cluttered.